docs/section22/Rule22-42.md

```diff
@@ -0,0 +1,98 @@
+
+            - create a dictionary of the expected capacities for use in the power validation check: `given_capacities = {}`
+            - the following lines or logic do the capacity validation check:
+            - loop through the expected_cwt_temps: `for cwt in expected_cwt_temps:`
+                - loop through each expected_ect_temps: `for ect in expected_ect_temps:`
+                    - look for the capacity validation point in capacity_validation_pts_dict: `if capacity_validation_pts_dict[[cwt, ect]]:`
+                        - calculate the expected capacity by multiplying the result of the formula by the rated_capacity: `expected_capacity = (cap_f_t_coefficients[0] + cap_f_t_coefficients[1] * cwt + cap_f_t_coefficients[2] * cwt^2 + cap_f_t_coefficients[3] * ect + cap_f_t_coefficients[4] * ect^2 + cap_f_t_coefficients[5] * cwt * ect) * rated_capacity`
+                        - get the given capacity: `given_capacity = capacity_validation_pts_dict[[cwt, ect]].result`
+                        - if the expected capacity matches the given capacity, add the expected capacity to the expected_capacities dictionary.  This should not be an exact match, but with a margin of error (see notes at the bottom for suggested margins of errors): `if expected_capacity == given_capacity: given_capacities[[cwt,ect]] = given_capacity`
+                        - otherwise, if the expected capacity doesn't match the given capacity, then change capacity_validation_status and go to Rule Assertion: `else: capacity_validation_status = "Expected capacity does not match given capacity at CHWT: " + cwt + " & ECT: " + ect; GO TO RULE ASSERTION`
+                    - otherwise this value doesn't exist, set the capacity_validation_status and go to rule assertion: `else: capacity_validation_status = "Required capacity validation point is missing.  Expected CHWT: " + cwt + ", expected ECT: " + ect; GO TO RULE ASSERTION`
+
+            - the following lines or logic do the power validation check:
+            - loop through the expected_cwt_temps: `for cwt in expected_cwt_temps:`
+                - loop through each expected_ect_temps: `for ect in expected_ect_temps:`
+                    - look for the power validation points in power_validation_pts_dict: `if power_validation_pts_dict[[cwt, ect]]:`
+                        - we are expecting to see multiple validation points aligning with the expected_validation_plr.  Create a list of part load ratios that are given.  Later we'll compare this with the expected list to make sure that all points are given: `given_plrs = []`
+                        - look at each power validation point in the list: `for power_validation_point in power_validation_pts_dict[[cwt, ect]]:`
+                            - get the load: `load = power_validation_point.load`
+                            - get the given power: `given_power = power_validation_point.result`
```

Comment:
Same commetn as above regarding .result. This is not the object from the rmd so I am having trouble understanding how python would know what .result means sicne you are loopign through a dictionary you created. I am probably missing something.

Reply:
the dictionary has been updated to make this clearer